src/components/views/__tests__/UserProfilePage.test.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Router, Route } from "react-router";
+import { createMemoryHistory } from "history";
+import { render } from "@testing-library/react";
+import UserProfilePage from "../UserProfilePage";
+const history = createMemoryHistory();
+
+const renderWithRouter = Component =>
+  render(
+    <Router history={history}>
+      <Route component={Component} />
```

Comment:
This code is not proper/
I think you confused both file names together.
This was the file you was supposed to test on for this PR.